docs/interpreter/jdbc.md

```diff
@@ -0,0 +1,224 @@
+
+* Postgres
+* MySql
+* MariaDB
+* Redshift
+* Hive
+
+If someone else used another database please report how it works to improve functionality.
+
+### Create Interpreter
+
+When create a interpreter by default use PostgreSQL with the next properties:
+
+<table class="table-configuration">
+  <tr>
+    <th>name</th>
+    <th>value</th>
+  </tr>
+  <tr>
+    <td>common.max_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>default.driver</td>
+    <td>org.postgresql.Driver</td>
+  </tr>
+  <tr>
+    <td>default.password</td>
+    <td>`********`</td>
+  </tr>
+  <tr>
+    <td>default.url</td>
+    <td>jdbc:postgresql://localhost:5432/</td>
+  </tr>
+  <tr>
+    <td>default.user</td>
+    <td>gpadmin</td>
+  </tr>      
+</table>
+
+Is not necessary add jar driver to the classpath for PostgreSQL.
+
+#### Simple connection
+
+Before creating the interpreter it is necessary to add to the Zeppelin classpath the path of the JDBC you want to use, to do it you must edit the file `zeppelin-daemon.sh` as shown in the table:
+
+```
+# Add jdbc connector jar
+ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/mysql-connector-java-5.1.6.jar"
+```
+
+For create the interpreter you need to specify connection parameters as shown in the table.
+
+<table class="table-configuration">
+  <tr>
+    <th>name</th>
+    <th>value</th>
+  </tr>
+  <tr>
+    <td>common.max_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>default.driver</td>
+    <td>driver name</td>
+  </tr>
+  <tr>
+    <td>default.password</td>
+    <td>`********`</td>
+  </tr>
+  <tr>
+    <td>default.url</td>
+    <td>jdbc url</td>
+  </tr>
+  <tr>
+    <td>default.user</td>
+    <td>user name</td>
+  </tr>      
+</table>
+
+#### Multiple connections
+
+This JDBC interpreter also allows connections to multiple data sources. For every connection is necessary a prefix for reference in the paragraph this way `%jdbc(prefix)`. Before creating the interpreter it is necessary to add to the Zeppelin classpath all paths to access to each driver's jar file you want to use, to do it you must edit the file `zeppelin-daemon.sh` as following:
+
+```
+# Add jdbc connector jar
+ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/RedshiftJDBC41-1.1.10.1010.jar"
+ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/mysql-connector-java-5.1.6.jar"
+```
+You can add all the jars you need to make multiple connections into the same interpreter. To create the interpreter you must specify the parameters, for example  we will create two connections to PostgreSQL and Redshift, the respective prefixes are `default` and `redshift`:
+
+<table class="table-configuration">
+  <tr>
+    <th>name</th>
+    <th>value</th>
+  </tr>
+  <tr>
+    <td>common.max_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>default.driver</td>
+    <td>org.postgresql.Driver</td>
+  </tr>
+  <tr>
+    <td>default.password</td>
+    <td>`********`</td>
```

Comment:
Hi @vgmartinez. Thank you for your effort to JDBC docs : )

If you purposed a code block for `********`, you have to use `<code>` tag in the table tag instead of the back quatation mark . Or, it doesn't work.